src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj

```diff
@@ -26,221 +26,6 @@
   </ItemGroup>
   <ItemGroup Label="Linked Files">
     <Compile Remove="Storage\Sqlite\**\*.cs" Condition="'$(DotNetBuildFromSource)' == 'true'" />
-    <Compile Include="..\..\..\CodeStyle\Core\CodeFixes\FixAllContextHelper.cs" Link="CodeFixes\FixAllOccurrences\FixAllContextHelper.cs" />
-    <Compile Include="..\..\..\Compilers\Shared\DesktopAnalyzerAssemblyLoader.cs">
```

Comment:
These linked files for utilities that are shared with compiler layer are also moved to shared `CompilerExtensions.shproj` shared project.